release-notes/major12/12.0.0/valtimo-backend-libraries.md

```diff
@@ -18,6 +18,14 @@ The following features were added:
   The assignees of a user task are now saved in the database by their user ID. When implementations use keycloak, the
   assignee column from the task table in the databases is automatically migrated from email to user ID.
 
+* **New endpoint to retrieve Zaaktypen**
+  A new endpoint has been added to the zaken-api module to retrieve zaaktypen: `/api/management/v1/zgw/zaaktype`. 
+  This replaces the `/api/v1/openzaak/zaaktype` endpoint from the openzaak module, which is now deprecated.
```

Comment:
Can you also mention this under deprecations?

Reply:
added